osgi/jta/src/main/resources/jbossts-properties.xml

```diff
@@ -90,7 +98,6 @@
     -->
     <entry key="RecoveryEnvironmentBean.recoveryModuleClassNames">
         com.arjuna.ats.internal.arjuna.recovery.AtomicActionRecoveryModule
-        com.arjuna.ats.internal.txoj.recovery.TORecoveryModule
```

Comment:
What is it used for ? I don't have those classes in the OSGi bundle at the moment.
I can add those though.